cysync/src/pages/mainApp/deviceUpdater/initialFlow/stepperFormComponents/upgradingDevice.tsx

```diff
@@ -151,6 +162,11 @@ const UpgradingDevice: React.FC<StepComponentProps> = ({ handleClose }) => {
       setTimeout(onClose, 350);
     }
   }, [isCompleted]);
+
+  const isAuthFailed =
+    errorResolutionState ===
+    DeviceUpgradeErrorResolutionState.DEVICE_AUTH_REQUIRED;
```

Comment:
Wont declaring this check as a variable be a problem as its dependent on a state?  
If we're sure its not a problem, reuse it above on the `refreshComponent` function